.devcontainer/download-builds.ts

```diff
@@ -1,43 +1,16 @@
 import { execSync } from 'child_process';
 import util from 'util';
-
+import {
+  getJobsByWorkflowId,
+  getPipelineId,
+  getWorkflowId,
+} from '../.github/scripts/shared/circle-artifacts';
 const exec = util.promisify(require('node:child_process').exec);
 
-function getGitBranch() {
-  const gitOutput = execSync('git status').toString();
-
-  const branchRegex = /On branch (?<branch>.*)\n/;
-  return gitOutput.match(branchRegex)?.groups?.branch || 'main';
-}
-
-async function getCircleJobs(branch: string) {
-  let response = await fetch(
-    `https://circleci.com/api/v2/project/gh/MetaMask/metamask-extension/pipeline?branch=${branch}`,
-  );
-
-  const pipelineId = (await response.json()).items[0].id;
-
-  console.log('pipelineId:', pipelineId);
-
-  response = await fetch(
-    `https://circleci.com/api/v2/pipeline/${pipelineId}/workflow`,
-  );
-
-  const workflowId = (await response.json()).items[0].id;
-
-  console.log('workflowId:', workflowId);
-
-  response = await fetch(
-    `https://circleci.com/api/v2/workflow/${workflowId}/job`,
-  );
-
-  const jobs = (await response.json()).items;
-
-  return jobs;
-}
-
```

Comment:
:information_source:  this logic is now in circle-artifacts file, as it's shared with the validate-page-object scriptl, as per @HowardBraham 's recommendation